webmidi/META.yml

```diff
@@ -0,0 +1,4 @@
+spec: https://webaudio.github.io/web-midi-api/
+suggested_reviewers:
+  - cwilso
```

Comment:
@cwilso @jussi-kalliokoski, I've sent org invites which you can accept at https://github.com/web-platform-tests. If you don't want to review tests for Web MIDI in this repo, can you suggest some good candidates?

Reply:
I'll review.  Sorry, been kinda backed up.